flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroRowDeserializationSchema.java

```diff
@@ -218,7 +231,7 @@ private Object convertAvroType(Schema schema, TypeInformation<?> info, Object ob
 		switch (schema.getType()) {
 			case RECORD:
 				if (object instanceof IndexedRecord) {
-					return convertAvroRecordToRow(schema, (RowTypeInfo) info, (IndexedRecord) object);
+					return convertAvroRecordToRow(schema, (RowTypeInfo) info, (IndexedRecord) object, jodaConverter);
```

Comment:
we can simply access the member field instead of passing it through the methods, no?

Reply:
Right in this class we can. I was too fast to follow the same scheme as in the AvroRowDataDeserializationSchema were we have runtime converters instead of member methods.